com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -95,22 +135,32 @@ var FileLoader = {
                 }
             }
         };
+        request.onretry = function(xhr, event, loadedSize, currentAttempt) {
+            onretry(loadedSize, currentAttempt);
+        }
         request.send();
     }
 };
 
 
 var EngineLoader = {
-    wasm_size: 2000000,
-    wasmjs_size: 250000,
-    asmjs_size: 4000000,
+    wasm_size: {{ DEFOLD_WASM_SIZE }},
+    wasmjs_size: {{ DEFOLD_WASMJS_SIZE}},
+    asmjs_size: {{ ASMJS_SIZE }},
+    wasm_instantiate_progress: 0,
+
+    stream_wasm: "{{html5.wasm_streaming}}" === "true",
 
-    stream_wasm: false,
+    updateWasmInstantiateProgress: function(totalDownloadedSize) {
+        EngineLoader.wasm_instantiate_progress = totalDownloadedSize * 0.1;
```

Comment:
Calculate what amount of total downloadable size is 10% and use if as part of progress that indicates wasm instantiating.